### DISCUSSION TITLE
Test duplicate custom action

### DISCUSSION THREAD

**Discussion by LorenzoAlu** [orchidsoftware/platform#2186] [Q&A]
Hi everyone :) 
I have a question.
I create a custom action to duplicate more Article Model in one go and that works great but I've a problem. 
I want to test this feature with PHPUNIT, someone can help me? I've no idea.

Thank you ;)

**Comment by tabuna** [comment]
Hello, take a look at this [document](https://orchid.software/en/docs/testing/). Most likely, you can do this using the steps indicated there.
